### DISCUSSION TITLE
Open Data: Geospatial Data Templates

### DISCUSSION THREAD

**Discussion by MahmoudAlaaOmran** [TDRA-ae/aegov-dls#9] [Q&A] [answered]
Hello,

Are there any predefined templates for the Geospatial data platform for us to refer to? If you have any sample data, please share it with us. It will be highly appreciated.

**Comment by moiz-conceptlz** [comment, marked as answer]
There is no particular template for maps and visualisations of mapping widgets. However, let's take an example of the suggested geospatial data platforms listed on the U.AE portal, and let's take the [HealthCare example](https://atlas.fgic.gov.ae/uaeatlas/Health/HealthcareHistory) on Union Atlas.

Now should you have the ability to style the widget and the visualisation of the map tool, and if you want to create a consistent format for buttons used on the zoom-in and zoom-out elements, then you could choose to follow the [button component](https://designsystem.gov.ae/docs/components/button) and specifically the section that describes "**Only Icons**".

Similarly, if you need to make other adjustments to the elements within the map widget, you may relate to existing components and their style. 

Hope this helps!
